source/api/cypress-api/config.md

```diff
@@ -9,6 +9,12 @@ title: Cypress.config
 {% url 'Read about configuration first.' configuration %}
 {% endnote %}
 
+{% note warning Scope %}
+Configuration set using `Cypress.config` _is only in scope for the current spec file._
+
+Cypress runs each spec file in isolation: the browser is exited between specs. Configuration changed in one spec won't be visible in other specs.
+{% endnote %}
```

Comment:
I expended the scope of this ticket because the same scoping applies to `Cypress.config` too.

> Somewhere else I read a comment from Brian Mann about being thorough when updating documentation... since this is so closely related I've included it under the rationale of _being thorough._

I also tested this locally to ensure that the scoping behaviour is the same as `Cypress.env`.